File: src/games/click.py
"""Click activity - collect every themed picture with a single click."""

from __future__ import annotations

from .targets import TargetActivity


class ClickActivity(TargetActivity):
    id = "click"
    title = "Single click"
    background = "click"
    themes_suffix = "click"
    require_double_click = False

File: web/js/games/click.js
// Click activity -- collect every themed picture with a single click.

import { TargetActivity } from "./targets.js";

export class ClickActivity extends TargetActivity {
  static id = "click";
  static suffix = "click";
  static bg = "click";
  static title = "Single click";
  static requireDouble = false;
}
